src/hooks/useDefaultLookup/useDefaultLookup.js

```diff
@@ -0,0 +1,86 @@
+import {
+  useContext,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+
+import {
+  AuthoritiesSearchContext,
+  useFacets,
+} from '@folio/stripes-authority-components';
+
+import {
+  DEFAULT_FILTERS,
+  DEFAULT_LOOKUP_OPTIONS,
+} from '../../constants';
+
+const getNonZeroDefaultFilters = (facets = {}, defaultTagFilters) => {
+  return Object.keys(facets).reduce((acc, facet) => {
+    const nonZeroFilters = defaultTagFilters[facet].filter(defaultFilterId => {
+      return facets[facet].values.some(filter => filter.id === defaultFilterId);
+    });
+
+    if (nonZeroFilters.length) {
+      acc[facet] = nonZeroFilters;
+    }
+
+    return acc;
+  }, { ...DEFAULT_FILTERS });
+};
+
+const useDefaultLookup = (cqlQuery, tag) => {
+  const {
+    filters,
+    searchIndex,
+    setSearchIndex,
+    setFilters,
+    setSearchDropdownValue,
+    searchDropdownValue,
+  } = useContext(AuthoritiesSearchContext);
+
+  const [areStatesUpdated, setAreStatesUpdated] = useState(false);
+
+  const {
+    qindex,
+    filters: defaultTagFilters,
+  } = DEFAULT_LOOKUP_OPTIONS[tag];
+
+  const { isLoading: areFacetsLoading, facets, refetch: fetchFacets } = useFacets({
```

Comment:
@Dmytro-Melnyshyn I understood 
> AND if the Authority source file facet has no LC Subject Headings (LCSH) value 
> THEN do not select an Authority source file facet value

as "if there is no option LC Subject Headings (LCSH) among the Authority source file facet - then don't select it"
Let's confirm with Khalilah which one it is, because if it's just ignoring selected value when there is no option - we don't have to pre-fetch facets to check for 0 records